Frontend/ui-customer/src/Screens/RideBookingFlow/HomeScreen/Transformer.purs

```diff
@@ -68,8 +70,18 @@ getLocation predcition = {
   , alpha : 1.0
   , fullAddress : dummyAddress
   , locationItemType : Just PREDICTION
+  , distance : getFormattedDistance (fromMaybe 0 (prediction ^._distance))
+  , showDistance : checkShowDistance (fromMaybe 0 (prediction ^._distance))
 }
 
+getFormattedDistance :: Int -> Maybe String
```

Comment:
we can't use that function because that function  is taking state as parameter and returning at_pickup/ at_drop in case of distance less than 10 meters